### PR TITLE
Perform async session query without workspace id and environment name

### PR DESCRIPTION
**Pull request by eugeneantonevich**

### CODE REVIEW

File: src/consts/request.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const RequestHeaderNotSet: undefined = undefined;
+export const RequestHeaderIgnored: null = null;
```
Comment (Selnapenek) [inline src/consts/request.ts:2]:
```suggestion
export const REQUEST_HEADER_NOT_SET: undefined = undefined;
export const REQUEST_HEADER_IGNORED: null = null;
```

File: src/common/execute.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export const executeAsync = async (
       await context.request(
         GraphqlActions.asyncSessionStatus,
         { sessionId },
-        { customEnvironment: DEFAULT_ENVIRONMENT_NAME },
+        { customEnvironment: RequestHeaderIgnored, customWorkspaceId: RequestHeaderIgnored },
```
Comment (Smirennikov8base) [inline src/common/execute.ts:27]: migration commit by token auth works if send here the options param

it is works if `customEnvironment` not matter: 
`result = (await context.request(GraphqlActions.asyncSessionStatus, { sessionId }, options)).status`;

**Review by Smirennikov8base — CHANGES_REQUESTED** [review]

File: src/common/execute.ts (hunk 2 above)
Comment (Smirennikov8base) [inline src/common/execute.ts:27]: options

File: src/consts/request.ts (hunk 1 above)
Comment (eugeneantonevich) [reply src/consts/request.ts:2]: +

File: src/common/execute.ts (hunk 2 above)
Comment (eugeneantonevich) [reply src/common/execute.ts:27]: I suppose, because we use token token. We have to ignore it in the request, too

Comment (Smirennikov8base) [reply src/common/execute.ts]: I have some problem still

image

File: src/common/execute.ts [hunk 3]
```diff
@@ -24,7 +24,11 @@ export const executeAsync = async (
       await context.request(
         GraphqlActions.asyncSessionStatus,
         { sessionId },
-        { customEnvironment: DEFAULT_ENVIRONMENT_NAME },
+        {
+          customEnvironment: REQUEST_HEADER_IGNORED,
+          customWorkspaceId: REQUEST_HEADER_IGNORED,
+          customAuthorization: REQUEST_HEADER_IGNORED,
+        },
```
Comment (Smirennikov8base) [inline src/common/execute.ts:31]: I have some problem still

image

**Review by Smirennikov8base — COMMENTED** [review]
not working still

File: src/common/execute.ts (hunk 3 above)
Comment (Smirennikov8base) [reply src/common/execute.ts:31]: @eugeneantonevich so now it  error shows for any type login

**Pull request merged**